src/AzureFunctions/azurefunctions.targets

```diff
@@ -1,13 +1,43 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Package">
+	<PropertyGroup>
+		<OpenAPIDefinitionsPath>$(RestSpecFolder)</OpenAPIDefinitionsPath>
+		<OpenAPIDefinitionOutputFile>$(DeployFullPath)\azuredeploy.api.yaml</OpenAPIDefinitionOutputFile>
+			<AnyRestService>false</AnyRestService>
+	</PropertyGroup>
+	 <ItemGroup>
+    	<DirsToClean Include="$(OpenAPIDefinitionsPath)" />
+ 	 </ItemGroup>
 
+	<ItemGroup>
+		<AllSelectedObjects Include="@(Object->HasMetadata('Selected')->WithMetadataValue('Selected','true'))" />
+	 </ItemGroup>
+
+	<Target Name="OpenAPIGeneration" Condition = "$(AZURE_FUNCTIONS_TRIGGER_TYPE) == 'http' AND $(APIRestSupport) != 'true'">
+		<Message Text="Generating Open API for Deployment" Importance="high"/>   
+
+		<RemoveDir Directories="@(DirsToClean)" ContinueOnError="true"/>         
+		<OpenKnowledgeBase Directory="$(KBPath)"  />
+
+		<SetGeneratorProperty Name="GENERATE_OPEN_API" Value="Yes"/>
+    	<SetGeneratorProperty Name="OPEN_API_VERSION" Value="2.0"/>
+    	<SpecifyOpenAPI ObjectName="%(Object.Type):%(Object.QualifiedName)" Condition="'%(Object.Selected)' == 'true' AND '%(Object.IsRest)' == 'true' And ('%(Object.IsSD)' != 'true')"/>
+		<SetGeneratorProperty Name="GENERATE_OPEN_API" Value="No"/>
```

Comment:
Just a note for later..

We should restore Generate Open API to previous value instead of turning off

Reply:
Done. Could you please review again?